src/App.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import "./App.css";
+import Block from './Block';
+import blocks from './block.json';
+
+
+export default class App extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.handleclick = this.handleclick.bind(this);
+
+    this.state = {
+      activeIndex: null,
+      activeChevron: '',
+    };
+  }
+
+
+  handleclick = (itemsId) => {  
+    return itemsId===this.state.activeIndex ? 
+    this.setState({ activeIndex: null, activeChevron: ''}): 
+    this.setState({ activeIndex: itemsId, activeChevron: 'active', });
```

Comment:
Could be optimised :) 
`const newState = itemsId===this.state.activeIndex ? { activeIndex: null, activeChevron: ''} : { activeIndex: itemsId, activeChevron: 'active', };
this.setState(newState);
`